test/integration_test/health_monitor_test.go

```diff
@@ -115,3 +119,87 @@ func stopKubeletTest(t *testing.T) {
 	destroyAndWait(t, ctxs)
 
 }
+
+func healthCheckFixTest(t *testing.T) {
+	// When a node's storage is offline stork should not bounce pods right away.
+	// It now waits for a minute and checks again to see if the storage driver is still offline.
+	// Bringing back node's storage within a minute should not affect anything
+	ctxs, err := schedulerDriver.Schedule(generateInstanceID(t, "stopdrivertest"),
+		scheduler.ScheduleOptions{AppKeys: []string{"mysql-1-pvc"}})
+	require.NoError(t, err, "Error scheduling task")
+	require.Equal(t, 1, len(ctxs), "Only one task should have started")
+
+	err = schedulerDriver.WaitForRunning(ctxs[0], defaultWaitTimeout, defaultWaitInterval)
+	require.NoError(t, err, "Error waiting for pod to get to running state")
+
+	// Get uuid for the app
+	var preUIDList, postUIDList []string
```

Comment:
Better to keep a map of podName->podUID. Adding to a map is more efficient than appending to a slice in golang